README.md

```diff
@@ -1,12 +1,20 @@
-## Run locally
-
-1. `composer install`
-1. `cp .env.example .env` and configure `.env` file
-1. `yarn install`
-1. `yarn run dev`
-1. Optional: make the search work:
+## Local development
+
+1. Install [Valet](https://laravel.com/docs/9.x/valet)
```

Comment:
We should indicate in the first step that we need PHP to work locally as well as a web server. We should also say that, for Mac owners, the preferred way is Valet. 

Most of the contributors of the old codebase don't have MacOS.

Reply:
Can you explain that please, not familiar enough with PHP to get into the nuances.